airbyte-cdk/python/airbyte_cdk/sources/abstract_source.py

```diff
@@ -24,8 +22,10 @@
 from airbyte_cdk.sources.source import Source
 from airbyte_cdk.sources.streams import Stream
 from airbyte_cdk.sources.streams.http.http import HttpStream
-from airbyte_cdk.sources.utils.schema_helpers import InternalConfig, split_config
-from airbyte_cdk.sources.utils.transform import TypeTransformer
+from airbyte_cdk.sources.utils.record_helper import \
+    stream_data_to_airbyte_message
+from airbyte_cdk.sources.utils.schema_helpers import InternalConfig, \
```

Comment:
did a formatter change? i don't recall seeing this before

Reply:
nope. Ran gradlew format. I'll look into what's wrong with my setup